src/Enterspeed.Cli/Services/FileService/SchemaFileService.cs

```diff
@@ -23,7 +27,7 @@ public SchemaFileService(ILogger<SchemaFileService> logger)
         _logger = logger;
     }
 
-    public void CreateSchema(string alias, SchemaType schemaType, string content = null)
+    public void CreateSchema(string alias, SchemaType schemaType, MappingSchemaVersion version = null)
```

Comment:
The CreateSchema method does not create empty schemas locally, like the old version, which means the the "schema create" command is failing in my test.

Reply:
This has been worked on. Could you verify that it works?

Reply:
Perfect, it's working.